tasks/service-runner/app/service/service-runner.ts

```diff
@@ -55,11 +55,11 @@ function _getStacCatalogs(dir: string): string[] {
  */
 function _getErrorMessage(logStr: string): string {
   // expect JSON logs entries
-  const regex = /\{"message":.*?"application":.*?\}/g;
-  const matches = logStr.match(regex);
+  const regex = /\{.*?\}/gs;
```

Comment:
Do you see this being a problem if we have a large number of messages logged by the backend service?

Reply:
Thats a non-greedy regex, so it will match the first '{' to the first following '}'. So it should get one valid log entry. I have added a `try/catch` around this anyway in case the extracted match is not valid JSON for some reason.